dallasnews/dmn-vendor-wrapper
# Fix footer links

## Description

This pr makes the links in the footer match the links in the main page. It does not address the font of the links.


## Testing and QA

- Download the branch
- npm run start
- go to http://localhost:8080/
- Open dallasnews.com in separate window
- Compare the footers and make sure that the link lists match
